### PR TITLE
Add end_use_control to good-on-application level

### PR DESCRIPTION
**Pull request by richtier**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,6 +39,15 @@ Service for handling backend calls in LITE.
 - At this point you might want to seed your database with some static
   - run `docker-compose run ./manage.py seedall`
 
+## Add a single user:
+
+Run the following command to add new users:
+
+```
+INTERNAL_USERS='[{"email"=>"foo@bar.gov.uk"}]' ./manage.py seedinternalusers
```
Comment (r4vi) [inline README.md:47]: does this format work? I always do `'[{"email":"foo@bar.gov.uk", "role": "Super User"}]'`

File: api/applications/models.py [hunk 2]
```diff
@@ -296,9 +297,13 @@ class AbstractGoodOnApplication(TimestampableModel):
     is_good_controlled = models.BooleanField(default=None, blank=True, null=True)
     comment = models.TextField(help_text="control review comment", default=None, blank=True, null=True)
     report_summary = models.TextField(default=None, blank=True, null=True)
-
     application = models.ForeignKey(BaseApplication, on_delete=models.CASCADE, null=False)
     control_list_entries = models.ManyToManyField(ControlListEntry)
+    end_use_control = ArrayField(
```
Comment (r4vi) [inline api/applications/models.py:302]: can you have more than one end use?

**Review by r4vi — APPROVED** [review]
left some questions but no blockers

File: api/applications/models.py (hunk 2 above)
Comment (richtier) [reply api/applications/models.py:302]: Daniel thinks maybe. We can lock it down to single when we get a concrete answer :)

File: README.md (hunk 1 above)
Comment (richtier) [reply README.md:47]: I tied that and it did not work, but `=>` did...odd

**Pull request merged**